exercises/outcomes/GT/GT-ready/template.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<knowl mode="exercise" xmlns="https://spatext.clontz.org" version="0.2">
+    <intro><p>Choose the most appropriate response for each question.</p></intro>
+    <!-- {{#questions}} -->
+    <!-- {{#area_comp}} -->
+    <knowl>
+        <content>
+            <p>
+Find the area of the parallelogram with vertices
```

Comment:
I wonder if these would be better if phrased like "Find the area of the parallelogram spanned by the vectors $\left[\begin{array}{c}2 \\ 3 \end{array}{c}\right]$$ and $\left[\begin{array}{c}4 \\ 5 \end{array}{c}\right]$" given that  that context is how they engage with the parallelograms in the activities.  Either way I don't think this kind of activity suffers from the lack of images

Reply:
I don't think students have necessarily been asked before about a parallelogram determined by a pair of vectors. (Can't say "spanned" of course, that's a plane.) This is what we introduce/illustrate in GT1. Our RAP material for this is [Khan Academy](https://www.khanacademy.org/math/cc-sixth-grade-math/cc-6th-geometry-topic/cc-6th-parallelogram-area/v/intuition-for-area-of-a-parallelogram) which is just base and height.

Reply:
We can (and should!) change the RAP resources to align with what we are asking them to do.

Reply:
Okay, I misunderstood the scope then, since you went ahead and changed one of the other questions.

Tagging #204 so this discussion doesn't get lost when we get around to improving all of the RATs.

Reply:
Fair retort - I guess my rationale was "I'm just going to change stuff that has to be changed for the purpose of a CheckIt RAT". Thanks for approving and xreffing #204 so we revisit this later.